nx_gzip/nx_gzip.py.data/nx_gzip.yaml

```diff
@@ -1,8 +1,8 @@
-linuxsrc_url: 'https://git.kernel.org/torvalds/t/linux-5.10-rc2.tar.gz'
+linuxsrc_url: 'https://cdn.kernel.org/pub/linux/kernel/v5.x/linux-5.15.tar.gz'
 in_file: 'in_file'
 out_file: 'out_file'
 new_file: 'new_file'
-file_size: '5'
+file_size: '150'
```

Comment:
maybe these  values can be local changes as the user always has control over these via YAML parameter

Reply:
Correct, I have changed it in YAML file too. Since I was changing the .py file for other reasons I have made changes in .py too.